engine/src/hopeit/server/api.py

```diff
@@ -570,14 +587,18 @@ def _builtin_schema(type_name: str, event_name: str, datatype: type) -> dict:
         },
         "description": f"{event_name} {type_name} payload"
     }
+    if type_format is not None:
+        schema['properties'][event_name]['format'] = type_format
+    return schema
 
 
 TYPE_MAPPERS = {
-    str: partial(_builtin_schema, 'string'),
-    int: partial(_builtin_schema, 'integer'),
-    float: partial(_builtin_schema, 'number'),
-    bool: partial(_builtin_schema, 'boolean'),
-    list: _array_schema
+    str: partial(_builtin_schema, 'string', None),
+    int: partial(_builtin_schema, 'integer', None),
+    float: partial(_builtin_schema, 'number', None),
+    bool: partial(_builtin_schema, 'boolean', None),
+    list: _array_schema,
```

Comment:
[Optional] [New Request] Add datetime to supported types (this is used on query_args and fields)